pkg/steps/pod.go

```diff
@@ -17,12 +17,17 @@ import (
 	"github.com/openshift/ci-operator/pkg/junit"
 )
 
+const testSecretName = "test-secret"
+const testSecretDefaultPath = "/usr/test-secrets"
+
 type PodStepConfiguration struct {
 	As                 string
 	From               api.ImageStreamTagReference
 	Commands           string
 	ArtifactDir        string
 	ServiceAccountName string
+	SecretName         string
```

Comment:
why not pass the whole `Secret` struct here?

Reply:
Changed.